addons/dispatch/1.2.x/dispatch-4.yaml

```diff
@@ -53,6 +53,16 @@ spec:
     version: "1.2.0"
     values: |
       ---
+      dispatch:
+        image: mesosphere/dispatch:1.2.0
+        testImage: mesosphere/dispatch-helm-test:1.2.0
+      tekton-dashboard:
+        image:
+          dashboard: gcr.io/tekton-releases/github.com/tektoncd/dashboard/cmd/dashboard:v0.7.0
+      tekton:
+        controller:
```

Comment:
@gpaul Could you tag `docker.io/gpaul/controller-10a3e32792f33651396d02b6855a6e36@sha256:61ce6412470881e83180399a8ea116258865da20e612914734e6b8c09b5a31fd` and add it here?

Reply:
doh good catch

Reply:
fixed, i'm going to re-run the test now though

Reply:
tests are happy still